### PR TITLE
RHDEVDOCS-4220: Doc: Openshift pipelines should only run on Linux nod…

### PR DESCRIPTION
**Pull request by eromanova97**
**Version(s)**: CP to `pipelines-docs-1.10`, `pipelines-docs-1.11`, `pipelines-docs-1.12`, `pipelines-docs-1.13`

**Issue**: [RHDEVDOCS-4220](https://issues.redhat.com/browse/RHDEVDOCS-4220)

**Preview link**:  [Installing OpenShift Pipelines](https://69178--docspreview.netlify.app/openshift-pipelines/latest/install_config/installing-pipelines)

**SME review**: @concaf 

**QE review**: @ppitonak 

**Peer review**: @lahinson

### CODE REVIEW

**Comment by ekristova** [comment]
Hello @concaf @ppitonak could you PTAL at this PR and leave feedback? Thank you!

**Review by ppitonak — APPROVED** [review]

**Review by concaf — APPROVED** [review]

File: install_config/installing-pipelines.adoc [hunk 1]
```diff
@@ -18,6 +18,11 @@ This guide walks cluster administrators through the process of installing the {p
 * You have installed xref:../tkn_cli/installing-tkn.adoc#installing-tkn[{pipelines-shortname} (`tkn`) CLI] on your local system.
 * Your cluster has the link:https://docs.openshift.com/container-platform/latest/installing/cluster-capabilities.html#marketplace-operator_cluster-capabilities[Marketplace capability] enabled or the Red Hat Operator catalog source configured manually.
 
+[NOTE]
+====
+In a cluster with both Windows and Linux nodes, the {pipelines-title} can only run on Linux nodes.
```
Comment (lahinson) [inline install_config/installing-pipelines.adoc:23]:
```suggestion
In a cluster with both Windows and Linux nodes, the {pipelines-title} can run on only Linux nodes.
```
- Is the note missing the word "Operator" after {pipelines-title}? If not, I'd remove the "the" from before {pipelines-title}.
- Per IBM Style, place "only" as close to the word it modifies as possible: https://www.ibm.com/docs/en/ibm-style?topic=grammar-adverbs-only

**Review by lahinson — COMMENTED** [review]
@ekristova I left a couple of comments for your consideration. Thanks!

**Pull request merged**